### PR TITLE
KEYCLOAK-15886 Add health probe scripts to quarkus container

### PR DESCRIPTION
**Pull request by ASzc**
These scripts just call the smallrye health endpoints via curl. Nothing complicated.

### CODE REVIEW

**Review by slaskawi — APPROVED** [review]
LGTM @ASzc !

@miquelsi @pedroigor I'll give you a few days just in case you wanted to review this changes.

File: server-x/probes/liveness.sh [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/bin/bash -e
+curl -sSfo /dev/null --max-time 10 "http://localhost:8080/auth/health/live"
```
Comment (miquelsi) [inline server-x/probes/liveness.sh:2]: The Quarkus distribution is placing Keycloak directly at the root of the URL, so no `/auth` context is needed here. The same for readiness check.

Comment (ASzc) [reply server-x/probes/liveness.sh:2]: Ok. I'll fix that. Updating [the spec](https://github.com/keycloak/keycloak-community/pull/55/files) would be a good idea.

Comment (miquelsi) [reply server-x/probes/liveness.sh]: Thanks, I've added a comment to the spec PR. Approving.

**Review by miquelsi — APPROVED** [review]

**Review by slaskawi — APPROVED** [review]
LGTM!

**Comment by slaskawi** [comment]
Let's give @pedroigor a day or two to look at this. Then, I'll merge it.

**Review by pedroigor — APPROVED** [review]

**Comment by slaskawi** [comment]
Integrated, thanks @ASzc !

**Pull request merged**